.github/workflows/sync-config-schema.yaml

```diff
@@ -53,13 +67,14 @@ jobs:
           GH_ACCESS_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
 
       - name: Set up Go
-        if: ${{ steps.release.outputs.update_main == 'false' }}
+        if: ${{ steps.release.outputs.is_alpha_version == 'false' }}
         uses: actions/setup-go@v5
         with:
           go-version-file: go.mod
 
-      - name: Update main docs version on alpha or beta versions
-        if: ${{ steps.release.outputs.update_main == 'true' }}
+      - name: Update main docs version on beta or rc versions
+        # update docs "main" version only on beta or -rc
+        if: ${{ steps.release.outputs.is_stable_version == 'false' && steps.release.outputs.is_alpha_version == 'false' }}
```

Comment:
the other conditions should be like this no?

Reply:
no. vCluster docs are now versioned, so we have "main" version (for the latest beta / RC version) and a list of stable version where each one has dedicated docs (e.g. `v0.19`, `v0.20`, `v0.21`).
We update docs "main" version only on beta & RC versions. For full versions, there is a CI job in `vcluster-config` repository that updates the given stable version in docs.

Reply:
Makes sense! I misunderstood the implications of the difference!